### PR TITLE
SAR-9678 - Incorporated Entity Identification - Add new stub data to CompaniesHouseStub

### PR DESCRIPTION
**Pull request by robertom-binqua**

### CODE REVIEW

File: app/uk/gov/hmrc/incorporatedentityidentificationfrontend/testonly/stubs/controllers/CompaniesHouseStubController.scala [hunk 1]
```diff
@@ -25,75 +26,123 @@ import scala.util.matching.Regex
...
+  private val defaultUkRegisteredOfficeAddress = RegisteredOfficeAddress(
+    address_line_1 = "10 HIGH STREET",
+    address_line_2 = "address line 2",
+    care_of = "test name",
+    country = "United Kingdom",
+    locality = "TELFORD",
+    po_box = "123",
+    postal_code = Some("TF3 4ER"),
+    premises = "1",
+    region = "SHROPSIRE"
```
Comment (HarrNM) [inline app/uk/gov/hmrc/incorporatedentityidentificationfrontend/testonly/stubs/controllers/CompaniesHouseStubController.scala:66]: Missing a 'H' => Shropshire. Also, looking into the call we make it doesn't look like care_of, po_box and region are returned to us so maybe remove them incase Test and Release (who requested the data) complains. Maybe put Telford in address line 2 and then Shropshire can be region

File: app/uk/gov/hmrc/incorporatedentityidentificationfrontend/testonly/stubs/controllers/CompaniesHouseStubController.scala [hunk 2]
```diff
@@ -25,75 +26,123 @@ import scala.util.matching.Regex
...
-    val stubCioProfile = Json.obj(
-      companyNameKey -> stubCharityName,
-      companyNumberKey -> companyNumber,
-      dateOfIncorporationKey -> "",
-      registeredOfficeAddressKey -> Json.obj()
+  private def toFullCompanyProfile(companyNumber: String, companyNameSuffix: Int, address_line_1: String): CompanyProfile =
+    CompanyProfile(
+      company_name = "VAT PENALTY REFORM " + companyNameSuffix,
+      company_number = companyNumber,
+      date_of_creation = stubOldDateOfIncorporation,
```
Comment (HarrNM) [inline app/uk/gov/hmrc/incorporatedentityidentificationfrontend/testonly/stubs/controllers/CompaniesHouseStubController.scala:112]: I think this is the wrong date of incorporation

File: app/uk/gov/hmrc/incorporatedentityidentificationfrontend/testonly/stubs/controllers/CompaniesHouseStubController.scala (hunk 1 above)
Comment (robertom-binqua) [reply app/uk/gov/hmrc/incorporatedentityidentificationfrontend/testonly/stubs/controllers/CompaniesHouseStubController.scala:66]: ok

File: app/uk/gov/hmrc/incorporatedentityidentificationfrontend/testonly/stubs/controllers/CompaniesHouseStubController.scala (hunk 2 above)
Comment (robertom-binqua) [reply app/uk/gov/hmrc/incorporatedentityidentificationfrontend/testonly/stubs/controllers/CompaniesHouseStubController.scala:112]: ops!

File: app/uk/gov/hmrc/incorporatedentityidentificationfrontend/testonly/stubs/controllers/CompaniesHouseStubController.scala [hunk 3]
```diff
@@ -96,4 +105,52 @@ class CompaniesHouseStubController extends InjectedController {
       }
     }
   }
+
+  private def validCompanyProfile(companyNumber: String, companyNameSuffix: Int, address_line_1: String): CompanyProfile = {
+    val registeredOfficeAddressReceivedByIncorporationInformation = RegisteredOfficeAddress(
+      address_line_1 = "10 HIGH STREET",
+      address_line_2 = "address line 2",
+      country = "United Kingdom",
+      locality = "TELFORD",
```
Comment (HarrNM) [inline app/uk/gov/hmrc/incorporatedentityidentificationfrontend/testonly/stubs/controllers/CompaniesHouseStubController.scala:114]: Please could we add Telford as address_line_2 and then Shropshire as locality, just incase they need all fields

Comment (robertom-binqua) [reply app/uk/gov/hmrc/incorporatedentityidentificationfrontend/testonly/stubs/controllers/CompaniesHouseStubController.scala:114]: I was going to write to you about all the details .... also premises ... do we need??
Screenshot 2022-05-27 at 13 21 57

Comment (robertom-binqua) [reply app/uk/gov/hmrc/incorporatedentityidentificationfrontend/testonly/stubs/controllers/CompaniesHouseStubController.scala:114]: okay

**Review by HarrNM — APPROVED** [review]

**Pull request merged**